DirectEmployers/MyJobs
# Remove services.

## What's this PR do?
Removes all the places we were using services, except on Solr. It turns out that Solr is the only correct use-case for services, and the rest were just bad assumptions.

## How should this be manually tested?
Run Datadog locally and run a few imports. You should still be able to find all the information about timing in the traces, but they won't show up as a wonky service in the services list.

## Any background context you want to provide?
https://help.datadoghq.com/hc/en-us/articles/115004050063-Resource-Trace-doesn-t-show-up-under-correct-Service
https://help.datadoghq.com/hc/en-us/articles/115000702546-What-is-the-Difference-Between-Type-Service-Resource-and-Name-
